components/lib/crash/src/main/java/mozilla/components/lib/crash/Crash.kt

```diff
@@ -81,11 +93,12 @@ sealed class Crash {
 
         companion object {
             internal fun fromBundle(bundle: Bundle) = NativeCodeCrash(
-                bundle.getString(INTENT_MINIDUMP_PATH, null),
-                bundle.getBoolean(INTENT_MINIDUMP_SUCCESS, false),
-                bundle.getString(INTENT_EXTRAS_PATH, null),
-                bundle.getBoolean(INTENT_FATAL, false),
-                bundle.getParcelableArrayList(INTENT_BREADCRUMBS) ?: arrayListOf()
+                uuid = bundle.getString(INTENT_UUID) as String,
```

Comment:
Here I'll need to add `?: UUID.randomUUID().toString()`. We are also processing crashes from GV via `CrashHandlerService` and they do not contain any UUID, so in those cases we need to generate one.

Reply:
You'll be adding this in a separate change?

Reply:
Just added it to this PR. :)